webextensions/api/identity.json

```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/identity",
           "support": {
             "chrome": {
-              "version_added": "≤58"
+              "version_added": "29"
```

Comment:
✅ Implemented in https://github.com/chromium/chromium/commit/499d1d369ad8d87efec1f09834bddaaa93eeea56 ([28.0.1498.0](https://github.com/chromium/chromium/releases/tag/28.0.1498.0) before 28.0.1500) and promoted to stable in https://github.com/chromium/chromium/commit/7b7b186bb995cac8d9ef216b96be1a5718d8ac3e ([29.0.1544.0](https://github.com/chromium/chromium/releases/tag/29.0.1544.0) before 29.0.1547).